k8s/namespaces/reform-scan/reform-scan-notification-service/prod.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: reform-scan-notification-service
+  annotations:
+    fluxcd.io/automated: "true"
+    repository.fluxcd.io/java: java.image
+    filter.fluxcd.io/java: glob:prod-*
+    repository.fluxcd.io/java.smoketests: java.smoketests.image
+    filter.fluxcd.io/java.smoketests: glob:prod-*
+spec:
+  values:
+    java:
+      environment:
+        NOTIFICATIONS_CONSUME_TASK_DELAY_IN_MS: "600000"
+        PENDING_NOTIFICATIONS_TASK_DELAY_IN_MS: "1800000"
+      testsConfig:
+        memoryLimits: "3072Mi"
+        serviceAccountName: tests-service-account
+        environment:
+          TEST_URL: "http://reform-scan-notification-service-prod.service.core-compute-prod.internal"
```

Comment:
Not sure if this is a good idea to run tests against a load balanced url , we should do it on local dns url ?

Reply:
But nothing to do with this PR

Reply:
See https://github.com/hmcts/cnp-flux-config/blob/master/k8s/namespaces/rpe/draft-store-service/aat.yaml#L31. With your setting if cluster 00 is rolling in a new change and cluster 01 is still waiting ( flux / helm operator delays), you are testing on load balancer url,  which means traffic to go to cluster 01 as well (where new change is not deployed)

Reply:
I'll update it in a separate PR